### PR TITLE
fixed ts strict error of EntityValue.tsx

### PR DESCRIPTION
**Pull request by sunilyadav840**
fixed typescript strict error of EntityValue.tsx

### CODE REVIEW

File: src/ConfigContext.ts [hunk 1]
```diff
@@ -4,6 +4,22 @@ export enum Platform {
...
+  ARM_API_VERSION: "",
+  GRAPH_ENDPOINT: "",
+  GRAPH_API_VERSION: "",
+  ARCADIA_ENDPOINT: "",
+  ARCADIA_LIVY_ENDPOINT_DNS_ZONE: "",
+  JUNO_ENDPOINT: "",
+  GITHUB_CLIENT_ID: "",
+  hostedExplorerURL: "",
+  allowedJunoOrigins: [""],
+};
```
Comment (southpolesteve) [inline src/ConfigContext.ts:22]: Undo this change. Change the types instead.

Comment (sunilyadav840) [reply src/ConfigContext.ts]: Updated changes

**Review by southpolesteve — APPROVED** [review]
